Home_task_14_1.py

```diff
@@ -0,0 +1,43 @@
+class Phone:
+    phone_number: str = ''
+    _count_calls: int = 0
+
+    def change_number(self, phone_number: str) -> str:
+        print(phone_number, ' has changed')
+        return phone_number
+
+    def output_count_callings(self) -> int:
+        return self._count_calls
+
+    def count_callings(self) -> None:
```

Comment:
Також проблема з назвою.
Я б скоріше очікував отримати тут кількість дзвінків.
```suggestion
    def accept_call(self) -> None:
```